source/read/count.txt

```diff
@@ -0,0 +1,145 @@
+.. uses tutorial.rst
+
+.. _pymongo-count:
+
+===============
+Count Documents
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: number, amount, estimation
+
+Overview
+---------
+
+In this guide, you can learn how to get an accurate and estimated count of the
+number of documents in your collection.
```

Comment:
S: nit just because "get" sounds too informal
```suggestion
In this guide, you can learn how to retrieve an accurate and estimated count of the
number of documents in a collection.
```